lib/full-site-editing/edit-site-export.php

```diff
@@ -5,6 +5,39 @@
  * @package gutenberg
  */
 
+/**
+ * Parses wp_template content and injects the current theme's
+ * stylesheet as a theme attribute into each wp_template_part
+ *
+ * @param string $template_content serialized wp_template content.
+ *
+ * @return string Updated wp_template content.
+ */
+function _remove_theme_attribute_from_content( $template_content ) {
+	$has_updated_content = false;
+	$new_content         = '';
+	$template_blocks     = parse_blocks( $template_content );
+
+	foreach ( $template_blocks as $key => $block ) {
+		if ( 'core/template-part' === $block['blockName'] && isset( $block['attrs']['theme'] ) ) {
```

Comment:
I wonder if we should be checking if the  "theme" is equal to the current theme? It seems consistent with checking "empty"  on the other side.

Reply:
> It seems consistent with checking "empty" on the other side.

That's an interesting way of looking at it. I don't have strong opinions. If we want to check if theme is equal to current theme, what do you think about just referencing `wp_get_theme()->get_stylesheet()` instead of accepting $theme as a parameter?

Reply:
Add the check here 3639d46